server/routes/householdRouter.js

```diff
@@ -64,21 +64,30 @@ householdRouter.post("/login", (req, res) => {
 /**
 * GET: get all households
 * POST: create a new household
+* DELETE: !!!DANGER!!! delete all households
 */
 householdRouter.route('/')
     .get((req, res) => {
         Household.find({}, (err, households) => {
             if (err) {
                 return res.status(500).json(err)
             }
-            res.json(households)
+            res.status(200).json(households)
         })
     })
     .post((req, res) => {
         let household = new Household(req.body)
         household.save()
         res.status(201).json(household)
     })
+    .delete((req, res) => {
```

Comment:
Is there a reason we're implementing this?

Reply:
Mostly just for maintenance, especially on the test server, which had entries based on outdated schemas. I discussed with Samyak that this is actually p dangerous to have and it might be worth including these with some checks, but I'll get rid of them for now.

Reply:
Cool that sounds like a better idea, we can add extra stuff like this later.